src/components/SearchPanel.vue

```diff
@@ -4,112 +4,558 @@
+              @change = "changeInput(input)"
+              class   = "form-control"
+              :id     = "input.id"
+            />
           </div>
-          <div v-else-if="forminput.type === 'selectfield' || forminput.type === 'autocompletefield'" class="form-group text" v-disabled="isSelectDisabled(forminput)">
-            <search-panel-label :forminput="forminput"/>
-            <bar-loader v-if ="forminput.options.dependance" :loading="state.loading[forminput.options.dependance] || forminput.loading"/>
-            <select2 :forminput="forminput" :autocompleteRequest="autocompleteRequest" @select-change="changeInput"/>
+
+          <!-- AUTOCOMPLETE FIELD -->
+          <div
+            v-else-if  = "['selectfield', 'autocompletefield'].includes(input.type)"
+            class      = "text"
+            v-disabled = "isSelectDisabled(input)"
+          >
+            <label :for="input.id" class="search-label">
+              <span>{{ input.label || input.attribute }}</span>
+              <span class="skin-color">{{ getLabelOperator(input.operator)}}</span>
+            </label>
+            <bar-loader
+              v-if     = "input.dependance"
+              :loading = "state.loading[input.dependance] || input.loading"
+            />
+            <select
+              :ref       = "'search_select_' + input.id"
+              :name      = "input.attribute"
+              class      = "form-control"
+              :id        = "input.id"
+              v-disabled = "input.disabled || input.loading"
+            >
+              <option
+                v-for  = "opt in input.values"
+                :key   = "opt.value"
+                :value = "opt.value"
+              >
+                <span v-if="opt.value === allvalue" v-t="'sdk.search.all'"></span>
+                <span v-else>{{ opt.key }}</span>
+              </option>
+            </select>
           </div>
-           <div v-if="forminput.type === 'datetimefield'" class="form-group text" v-disabled="state.loading[forminput.options.dependance] || false">
-             <search-panel-label :forminput="forminput"/>
-             <search-datetime :forminput="forminput" @change="changeInput"/>
+
+          <!-- DATETIME FIELD -->
+          <div
+            v-else-if  = "'datetimefield' === input.type"
+            class      = "text"
+            v-disabled = "state.loading[input.dependance] || false"
+          >
+            <label :for="input.id" class="search-label">
+              <span>{{ input.label || input.attribute }}</span>
+              <span class="skin-color">{{ getLabelOperator(input.operator)}}</span>
+            </label>
+            <div class="input-group date">
+              <input :id="input.id" type='text' class="form-control" />
+              <span class="input-group-addon skin-color">
+                <span :class="g3wtemplate.getFontClass(input.options.format.time ? 'time': 'calendar')"></span>
+              </span>
+            </div>
```

Comment:
@Raruto to solve issue on datetimepiker click on calendar icon need to move id to div as follow
```suggestion
            <div :id="input.id" class="input-group date">
              <input type='text' class="form-control" />
              <span class="input-group-addon skin-color">
                <span :class="g3wtemplate.getFontClass(input.options.format.time ? 'time': 'calendar')"></span>
              </span>
            </div>
```

Reply:
@volterra79 Your suggestion triggers a console.warning.

what do you think about label element? ref: https://github.com/g3w-suite/g3w-client/pull/574/commits/397fda59f94d717d19c149ac682f3df88cab09b5

Reply:
> @volterra79 Your suggestion triggers a console.warning.
> 
> what do you think about label element? ref: [397fda5](https://github.com/g3w-suite/g3w-client/commit/397fda59f94d717d19c149ac682f3df88cab09b5)

reference documentation https://getdatepicker.com/4/ suggest to set id on div